metadata_store_scripts/bootstrap.sql

```diff
@@ -272,3 +272,276 @@ BEGIN
 END;
 -- source: V2024.08.25.0__add_conditional_masking_support
 ALTER TABLE discovered_ruleset ALTER COLUMN assigned_algorithm VARCHAR(MAX);
+
+-- source: V2024.10.24.0__add_checkpointing_and_logging
```

Comment:
Can we create a script that updates the bootstrap.sql script with every new migration script added for metadata store?

Reply:
Done